apps/studio/schemas/person.ts

```diff
@@ -0,0 +1,39 @@
+import { HeartIcon } from "@sanity/icons";
+import { defineField, defineType } from "sanity";
+
+export default defineType({
+  name: "person",
+  icon: HeartIcon,
+  title: "People",
+  type: "document",
+  fields: [
+    defineField({
+      name: "name",
+      title: "Name",
+      type: "string",
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+      name: "pronouns",
+      title: "Pronouns",
+      type: "string",
+    }),
+    defineField({
+      name: "linkedinURL",
+      title: "LinkedIn URL",
+      type: "url",
+    }),
+    defineField({
+      name: "profilePic",
+      title: "Profile Picture",
+      description: "Images should preferably have a 1:1 aspect ratio.",
+      type: "image",
```

Comment:
Consider adding
```ts
options: {
  hotspot: true
}
```
here to allow the user crop and preview the image.
https://www.sanity.io/docs/image-type#hotspot-3e6da78954a8

Reply:
Added in [be8cb05](https://github.com/HackAtUCI/hack.ics.uci.edu/pull/12/commits/be8cb056605ba8708d4ceafed6fa6af9e446787e)